scripts/PearlPowderISIS/pearl_routines.py

```diff
@@ -0,0 +1,1338 @@
+# pylint: disable=anomalous-backslash-in-string, global-variable-undefined, global-variable-not-assigned, too-many-lines
+# pylint: disable=invalid-name, too-many-arguments, superfluous-parens, too-many-branches, redefined-builtin
+
+import os.path
+import sys
+
+from mantid.simpleapi import *
+import numpy as n
+
+
+# directories generator
+import pearl_calib_factory
+import pearl_cycle_factory
+
+
+def PEARL_startup(usern="matt", thiscycle='11_1'):
+    global pearl_file_dir
```

Comment:
Yes, and not just one or two. That's what would worry me more than any other issue in this code. :disappointed: